app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt

```diff
@@ -46,10 +74,19 @@ fun PlotSurfacePreview() {
 
 @Composable
 fun Map(xPercent: Float, yPercent: Float, modifier: Modifier = Modifier) {
-    // TODO fill out the square map here.
-    //  Create a Box composable with a size of 300.dp
-    //  and then create an inner Box composable
-    //  with a clip shape of CircleShape.
+    Box(
+        modifier
+            .size(squareBox.dp)
+            .background(Color.Blue)
+    ) {
+        Box(
+            modifier
+                .offset((xPercent * 300 - 18).dp, (yPercent * 300 - 18).dp)
```

Comment:
Probably use the same box constant here, but other than that, lgtm.

Reply:
Thanks, updated